docs/data/osdf/install-cache-rpm.md

```diff
@@ -0,0 +1,125 @@
+title: Installing the OSDF Cache by RPM
+
+Installing the OSDF Cache
+=========================
+
+This document describes how to install a Pelican-based Open Science Data Federation (OSDF) Cache service via RPMs.
+This service allows a site or regional network to cache data 
+
+!!! note
+    The cache must be registered with the OSG prior to joining the data federation.
+    You may start the registration process prior to finishing the installation by [using this link](#registering-the-cache) 
+    along with information like:
+
+    * Resource name and hostname
+    * Administrative and security contact(s)
+
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following requirements:
+
+* __Operating system:__ A RHEL 8 or RHEL 9 or compatible operating systems.
+* __User IDs:__ If they do not exist already, the installation will create the Linux user named `xrootd` for running daemons.
+* __File Systems:__ The cache should have a partition of its own for storing data and metadata.
+* __Host certificate:__ Required for authentication.  See note below.
+* __Network ports:__ The cache service requires the following ports open:
+  * Inbound TCP port 8443 for file access via the HTTP(S) and XRoot protocols.
+  * (Optional) Inbound TCP port cache for access to the web interface for monitoring and configuration;
+    if enabled, this should be restricted to the LAN or management network.
+* __Hardware requirements:__ We recommend that an origin has at least 1Gbps connectivity and 12GB of RAM.
+  We suggest that several gigabytes of local disk space be available for log files,
+  although some logging verbosity can be reduced.
+
+As with all OSG software installations, there are some one-time steps to prepare in advance:
+
+* Obtain root access to the host
+* Prepare [the required Yum repositories](../../common/yum.md)
+
+
+!!! note "OSG 23"
+    In OSG 23, the Pelican-based OSDF RPMs are only available in the "osg-upcoming" repositories.
+
+!!! note "Host certificates"
+    Caches should use a CA that is accepted by major browsers and operating systems,
+    such as InCommon RSA or [Let's Encrypt](../../security/host-certs/lets-encrypt).
+    IGTF certs are not recommended because clients are not configured to accept them by default.
+    Note that you will need the full certificate chain, not just the certificate.
+
+    The following locations should be used (note that they are in separate directories):
+    
+    * **Host Certificate Chain**: `/etc/pki/tls/certs/pelican.crt`
+    * **Host Key**: `/etc/pki/tls/private/pelican.key`
+
+
```

Comment:
Since we've got a lot (?) of non-Pelican caches out there, it's probably worth having a separate, leading "upgrade section" here that outlines the overview of how to perform an upgrade. Maybe it's something as simple as "follow all the instructions here, turn off the old cache services, then register the new cache"

Or maybe we can't have two cache services running concurrently...